src/lib/index/wb_cache.cpp

```diff
@@ -241,6 +246,26 @@ folly::Future< bool > IndexWBCache::async_cp_flush(IndexCPContext* cp_ctx) {
 void IndexWBCache::do_flush_one_buf(IndexCPContext* cp_ctx, IndexBufferPtr buf, bool part_of_batch) {
     LOGTRACEMOD(wbcache, "cp {} buf {}", cp_ctx->id(), buf->to_string());
     buf->set_state(index_buf_state_t::FLUSHING);
+
+#ifdef _PRERELEASE
+
+    if (m_abrupt_flush.load() == true) {
```

Comment:
couldnt we add a function cp_ctx->isabrupt() ?

Reply:
OK. sure I ll add it to the cp_ctx instead of wbcache